to-do.md

```diff
@@ -1,5 +1,5 @@
-1. colocar arquivo de configuracao de eslint no projeto para manter padrao
-1. colocar arquivo de configuracao de prettier no projeto para manter padrao
-1. colocar arquivo de configuracao de vs code no projeto para sugerir extensions e configuracao de format file on save
-1. fazer com que os nomes das variaveis de ambiente customizadas aparecam no autocomlete ao digitar "process.env" em arquivos `.ts`
+1. colocar arquivo de configuracao de eslint no projeto para manter padrao ✅
+1. colocar arquivo de configuracao de prettier no projeto para manter padrao ✅
+1. colocar arquivo de configuracao de vs code no projeto para sugerir extensions e configuracao de format file on save ✅
+1. fazer com que os nomes das variaveis de ambiente customizadas aparecam no autocomlete ao digitar "process.env" em arquivos `.ts` ✅
```

Comment:
nice icons.. no related TODOs in the code, right?

Reply:
yeah, there are no TODOs related to this.